channel.json

```
@@ -39,6 +39,8 @@
     "https://raw.githubusercontent.com/usfbih8u/micro-gutter-message/master/repo.json",
     //gzplugin
     "https://raw.githubusercontent.com/dzmanto/gzplugin4micro/master/repo.json",
+    //iconize_tabs
```

Please use the name in your repo.json


```suggestion
    //iconic_tabs
```